hoomd/write/table.py

```diff
@@ -275,6 +277,30 @@ def act(self, timestep=None):
             self._write_row(output_dict)
             self.output.flush()
 
+    def __getstate__(self):
+        state = copy.copy(self.__dict__)
+        state.pop('_comm', None)
+        # This is to handle when the output specified is just stdout. By default
+        # file objects like this are not picklable, so we need to handle it
+        # differently. We let `None` represent stdout in the state dictionary.
```

Comment:
This is confusing. So objects using `stdout` are picklable, because that value is mapped to `None`. Are objects using other outputs (file outputs, stderr, etc.) not picklable?

Reply:
No there are not currently. It may be implementable to some extent, but I didn't attempt to do so here.